migration_toolkit_for_containers/index.adoc

```diff
@@ -0,0 +1,79 @@
+[id="migration-toolkit-for-containers-overview"]
+= Migration toolkit for containers overview
+include::modules/common-attributes.adoc[]
+:context: migration-toolkit-for-containers-overview
+
+toc::[]
+
+You can migrate stateful application workloads between {product-title} 4 clusters at the granularity of a namespace by using the Migration Toolkit for Containers (MTC). To learn more about MTC see xref:../migration_toolkit_for_containers/about-mtc.adoc#about-mtc[Understanding MTC].
+
+[discrete]
+[id='mtc-overview-install-mtc']
+=== MTC installation
```

Comment:
Why is this a level 3 header? I think it should be level 2 (==).

Reply:
Changed back to (==)